main.md

```diff
@@ -213,7 +211,7 @@ The format of the `request_uri` value is at the discretion of the authorization
 
 The `request_uri` MUST be bound to the client that posted the authorization request.
 
-Since the request URI can be replayed, its lifetime SHOULD be short and preferably limited to one-time use.
+Since parts of the request content, e.g. the `code_challenge` parameter value, is unique to a certain authorization request, a `request_uri` should be limited to one-time use.
```

Comment:
should this "should" be a "SHOULD"?

Reply:
also was it intentional or is it okay that it no longer saying anything about the lifetime being short?